src/plugins/default/markbind-plugin-anchors.js

```diff
@@ -0,0 +1,40 @@
+const cheerio = module.parent.require('cheerio');
+const md = require('./../../lib/markbind/src/lib/markdown-it');
+
+const ANCHOR_HTML = '<a class="fa fa-anchor" href="#"></a>';
+
+/**
+ * Generates a heading selector based on the indexing level
+ * @param headingIndexingLevel to generate
+ */
+function generateHeadingSelector(headingIndexingLevel) {
+  let headingsSelector = 'h1';
+  for (let i = 2; i <= headingIndexingLevel; i += 1) {
+    headingsSelector += `, h${i}`;
+  }
+  return headingsSelector;
+}
+
+/**
+ * Adds anchor links to headers
+ */
+module.exports = {
+  postRender: (content, pluginContext, frontMatter, pageConfig) => {
```

Comment:
Would it be possible to write unit tests for this?

Reply:
To clarify: This is the part where we think could be done in a separate PR. Btw, this is different from the functional test that we already have in `test_site`.

Reply:
Sure, I'll make an issue